docker/prowbazel/Dockerfile

```diff
@@ -33,6 +33,11 @@ WORKDIR ${HOME}
 # code and running bazel in different dir than the calling script.
 ENV TEST_TMPDIR /home/bootstrap/.cache/bazel
 
+# Move preloaded Bazel binaries to bootstrap's cache.
+RUN mkdir -p /home/bootstrap/.cache/
```

Comment:
Each `RUN` statement creates a new docker layer. I'd recommend trying to combine these as much as possible to reduce the number of layers we need to download and extract:

Running multiple commands in one layer (at least combine these three commands into one):
* https://github.com/kubernetes/test-infra/blob/1972ae8f994d61b88f7a84c9947aa88bcb3aae18/images/bootstrap/Dockerfile#L30-L52

Adding multiple env vars into one layer (if you want to do a more aggressive refactor):
* https://github.com/kubernetes/test-infra/blob/1972ae8f994d61b88f7a84c9947aa88bcb3aae18/images/bootstrap/Dockerfile#L21-L22

Reply:
Done & I don't.